src/Command/InstallWebDirCommand.php

```diff
@@ -73,6 +73,7 @@ protected function executeLocked(InputInterface $input, OutputInterface $output)
         $webDir = rtrim($baseDir, '/').'/web';
 
         $this->addFiles($webDir);
+        $this->removeInstallPhp($webDir);
```

Comment:
Why did you add this if you are not distributing the `install.php` file anymore?